gnat2goto/driver/tree_walk.adb

```diff
@@ -1432,6 +1433,22 @@ package body Tree_Walk is
       return SEE_Fun_Call;
    end Make_Assume_Expr;
 
+   -----------------------------
+   -- Do_Qualified_Expression --
+   -----------------------------
+
+   function Do_Qualified_Expression (N : Node_Id) return Irep is
+      Value : constant Irep := Do_Expression (Expression (N));
+      Type_Of_Val : constant Irep :=
+         Follow_Symbol_Type (
```

Comment:
Is following necessary here?

Reply:
Without it it wasn't getting the correct bounds.

Reply:
Without the follow you're getting a symbol type here, and `Make_Range_Assert` is expecting a resolved type instead